File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.ContentGrid.NUnit._VB/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.vb
﻿'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public class Tests
    '^^
    '{[{

    ' Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Test>
    Public Async Function EnsureSampleDataServiceReturnsContentGridDataAsync() As Task
        Dim actual = Await SampleDataService.GetContentGridDataAsync()

        Assert.AreNotEqual(0, actual.Count)
    End Function
    '}]}
End Class

File: templates/Uwp/_comp/Prism/Test.UnitTests.Core.SampleData.ContentGrid.xUnit/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.cs
﻿//{[{
using System.Threading.Tasks;
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.XUnit
{
    public class Tests
    {
        //^^
        //{[{

        // Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Fact]
        public async Task EnsureSampleDataServiceReturnsContentGridDataAsync()
        {
            var dataService = new SampleDataService();
            var actual = await dataService.GetContentGridDataAsync();

            Assert.NotEmpty(actual);
        }
        //}]}
    }
}
